SPECS/libepoxy/libepoxy.spec

```diff
@@ -1,20 +1,19 @@
 Summary:        epoxy runtime library
 Name:           libepoxy
-Version:        1.5.5
-Release:        4%{?dist}
+Version:        1.5.10
+Release:        1%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://github.com/anholt/libepoxy
-Source0:        %{url}/releases/download/%{version}/%{name}-%{version}.tar.xz
-
+Source0:        %{url}/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
 BuildRequires:  gcc
 BuildRequires:  libEGL-devel
 BuildRequires:  libGL-devel
 BuildRequires:  libX11-devel
 BuildRequires:  mesa-dri-drivers
 BuildRequires:  meson
-BuildRequires:  pkg-config
+BuildRequires:  pkgconfig
```

Comment:
Does this build successfully now? I wonder how it built successfully before the change if the pkgconfig name is the correct one.

Reply:
Yes, according to the pipeline, it built fine.
snambakam@mariner-2-g-1 [ ~ ]$ rpm -ql libepoxy
/usr/lib/libepoxy.so.0
/usr/lib/libepoxy.so.0.0.0
/usr/share/doc/libepoxy
/usr/share/doc/libepoxy/README.md
/usr/share/licenses/libepoxy
/usr/share/licenses/libepoxy/COPYING
snambakam@mariner-2-g-1 [ ~ ]$ rpm -ql libepoxy-devel
/usr/include/epoxy
/usr/include/epoxy/common.h
/usr/include/epoxy/egl.h
/usr/include/epoxy/egl_generated.h
/usr/include/epoxy/gl.h
/usr/include/epoxy/gl_generated.h
/usr/include/epoxy/glx.h
/usr/include/epoxy/glx_generated.h
/usr/lib/libepoxy.so
/usr/lib/pkgconfig/epoxy.pc
snambakam@mariner-2-g-1 [ ~ ]$ rpm -q libepoxy
libepoxy-1.5.10-1.cm2.x86_64